packages/tests/test/reya_common/BaseReyaForkTest.sol

```diff
@@ -55,13 +55,31 @@ contract BaseReyaForkTest is StorageReyaForkTest {
         );
     }
 
+    function roundPrice(UD60x18 price, UD60x18 priceSpacing, bool roundUp) internal pure returns (UD60x18) {
+        UD60x18 reminder = price.mod(priceSpacing);
+        UD60x18 roundedDown = price.sub(reminder);
+
+        if (reminder.eq(ZERO_ud) || !roundUp) {
+            return roundedDown;
+        }
+
+        return roundedDown.add(priceSpacing);
+    }
+
     function getMarketSpotPrice(uint128 marketId) internal view returns (UD60x18 marketSpotPrice) {
         MarketConfigurationData memory marketConfig = IPassivePerpProxy(sec.perp).getMarketConfiguration(marketId);
         NodeOutput.Data memory marketNodeOutput =
             IOracleManagerProxy(sec.oracleManager).process(marketConfig.oracleNodeId);
         return ud(marketNodeOutput.price);
     }
 
+    function getMarketSpotPrice(uint128 marketId, SD59x18 base) internal view returns (UD60x18 marketSpotPrice) {
```

Comment:
I'd be more inclined to add roundUp as boolean and use this function in general.